src/SFA.DAS.Admin.Roatp.Web/Controllers/ProviderStatusConfirmationController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Admin.Roatp.Domain.Models;
+using SFA.DAS.Admin.Roatp.Web.Infrastructure;
+using SFA.DAS.Admin.Roatp.Web.Models;
+
+namespace SFA.DAS.Admin.Roatp.Web.Controllers;
+
+
+[Route("providers/{ukprn}/status-confirmed", Name = RouteNames.ProviderStatusUpdateConfirmed)]
+public class ProviderStatusConfirmationController(IOuterApiClient _outerApiClient) : Controller
+{
+    [Authorize(Roles = Roles.RoatpAdminTeam)]
+
+    public async Task<IActionResult> Index(string ukprn, CancellationToken cancellationToken)
```

Comment:
Can the confirmation  endpoint be part of the status update controller... just to keep things coupled where it makes sense.